docs/core/extensions/httpclient-factory-troubleshooting.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Troubleshoot IHttpClientFactory issues
+description: Learn how to troubleshoot common HttpClient and IHttpClientFactory problems.
+author: CarnaViire
+ms.author: knatalia
+ms.date: 08/21/2024
+---
+
+# Common `IHttpClientFactory` usage issues
+
+In this article, you'll learn some of the most common problems you can run into when using `IHttpClientFactory` to create `HttpClient` instances.
+
+`IHttpClientFactory` is a convenient way to set up multiple `HttpClient` configurations in the DI container, configure logging, set up resilience strategies, and more. `IHttpClientFactory` also incapsulates the lifetime management of `HttpClient` and `HttpMessageHandler` instances, to prevent problems like socket exhaustion and losing DNS changes. For an overview on how to use `IHttpClientFactory` in your .NET application, you can check out [IHttpClientFactory with .NET](httpclient-factory.md).
```

Comment:
```suggestion
`IHttpClientFactory` is a convenient way to set up multiple `HttpClient` configurations in the DI container, configure logging, set up resilience strategies, and more. `IHttpClientFactory` also incapsulates the lifetime management of `HttpClient` and `HttpMessageHandler` instances, to prevent problems like socket exhaustion and losing DNS changes. For an overview on how to use `IHttpClientFactory` in your .NET application, see [IHttpClientFactory with .NET](httpclient-factory.md).
```